src/pages/map/components/PlaceBottomSheet.jsx

```
@@ -0,0 +1,137 @@
+import { useState, useRef, useEffect } from "react";
+import { motion, animate } from "framer-motion";
+import PlaceContent from "./PlaceContent";
+
+const PlaceBottomSheet = ({ place, onClose }) => {
+  const [liked, setLiked] = useState(place.liked || false);
+  const [height, setHeight] = useState(MIN_HEIGHT);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)